flake.nix

```diff
@@ -0,0 +1,169 @@
+{
+  description = "High-level tracing language for Linux eBPF";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/release-22.11";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    # We only support 64-bit linux systems
```

Comment:
I think it's reasonable to limit this to tested systems in the beginning, but I don't think this is true of bpftrace - it supports quite a few architectures.

Reply:
I think nix only supports aarch and x86 on linux and osx, right? We do support quite a few architectures but none 32 bit. And osx has no bpf support.

So that limits nix support to aarch64-linux and x86-64-linux, right? Or am I misunderstanding something?

Reply:
bpftrace does support 32-bit ARM now, FWIW